client/header/index.js

```diff
@@ -106,29 +93,10 @@ export const Header = ( { sections, isEmbedded = false, query } ) => {
 					onInstall={ dismissHandler }
 				/>
 			) }
-			<h1 className="woocommerce-layout__header-breadcrumbs">
-				{ sections.map( ( section, i ) => {
-					const sectionPiece = Array.isArray( section ) ? (
-						<Link
-							href={
-								isEmbedded
-									? getAdminLink( section[ 0 ] )
-									: getNewPath( {}, section[ 0 ], {} )
-							}
-							type={ isEmbedded ? 'wp-admin' : 'wc-admin' }
-							onClick={ trackLinkClick }
-						>
-							{ section[ 1 ] }
-						</Link>
-					) : (
-						section
-					);
-					return (
-						<span key={ i }>
-							{ decodeEntities( sectionPiece ) }
-						</span>
-					);
-				} ) }
+			<h1 className="woocommerce-layout__header-heading">
+				<Text variant="subtitle.small">
```

Comment:
Nice use of `Text`!

Reply:
Well thanks!

Reply:
One small issue with this is that text will by default produce a `p` tag which is semantically incorrect inside an `h1` tag.

There's an `as` property on `Text` we can use to simplify this a bit:

```js
			<Text
				className="woocommerce-layout__header-heading"
				variant="subtitle.small"
				as="h1"
			>
				{ decodeEntities( pageTitle ) }
			</Text>
```

Reply:
That simplifies all the things, I like it!